cmd/csi-sanity/main.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2017 Luis Pabón luis@portworx.com
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/kubernetes-csi/csi-test/pkg/sanity"
+)
+
+const (
+	prefix string = "csi."
+)
+
+var (
+	VERSION = "(dev)"
+	config  = sanity.NewTestConfig()
```

Comment:
This can be moved into `main()` now. (:

Reply:
Indeed :-) Done.

Reply:
@misterikkit can you also review PR #233? That needs to go in first.